BSP/STM32F413/Src/BSP_WDTimer.c

```diff
@@ -13,7 +13,7 @@ void BSP_WDTimer_Init(void) {
     // IWDG has a timeout value of 4.096 / 8 seconds (about 500 ms).
 	IWDG_WriteAccessCmd(IWDG_WriteAccess_Enable);       // Enable access to the IWDG_PR and IWDG_RLR registers
 	IWDG_SetPrescaler(IWDG_Prescaler_32);               // Prescaler divider feeding the counter clock
-	IWDG_SetReload(IWDG_RLR_RL / 8);                    // Reload value set to 0xFFF / 8
+	IWDG_SetReload(IWDG_RLR_RL / 4);                    // Reload value set to 0xFFF / 8
```

Comment:
I think I aked you, but just a reminder to comment why this change was made in the timer :)